doc/script_commands.txt

```diff
@@ -1963,18 +1967,31 @@ prontera,150,150,0	script	TestNPC	123,{
 	input(.@a);
 	input(.@b);
 	/* Function call */
-	mes(.@a+" + "+.@b+" = "+MyAdd(.@a, .@b));
+	mesf("%i + %i = %i", .@a, .@b, MyAdd(.@a, .@b));
 	close();
 
 	/* Function definition */
 	function MyAdd {
-		return(getarg(0)+getarg(1));
+		return (getarg(0) + getarg(1));
 	}
-}
 
 
 ---------------------------------------
 
+*<function name>({<arg>...})
+*"<npc name>"::<function name>({<arg>...})
+
+In its first form, calls a previously defined local function. In its second
+form, calls a previously defined exported local function of another NPC.
```

Comment:
`calls a previously defined exported local function of another NPC` … that's a mouthful,
but I don't really know how to explain it in simpler terms